solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -72,6 +72,11 @@ contract WalletRegistry is IRandomBeaconConsumer, IWalletRegistry, Ownable {
     ///         operator affected.
     uint256 public maliciousDkgResultNotificationRewardMultiplier;
 
+    /// @notice Duration of the sortition pool rewards ban imposed on operators
+    ///         who missed their turn for DKG result submission or who failed
+    ///         a heartbeat.
+    uint256 public sortitionPoolRewardsBanDuration;
```

Comment:
Shouldn't we set the initial value here or in the constructor?

Reply:
Yeah, we should. Fixed in https://github.com/keep-network/keep-core/pull/2868/commits/2642b128b76751959c527cf17ae5ba75fcefa70e.